File: aufgaben/lf05/22-10-2024/einfaches-gewinnspiel.py
eingabe = 0
while eingabe != 7:
    eingabe = int(input("Bitte gib eine Zahl zwischen 1 - 10 ein! "))
    if eingabe == 7:
        print("Herzlichen Glückwunsch, du hast richtig geraten!")
    else:
        print("Leider falsch, versuche es erneut.")

File: beispiele/einfaches-gewinnspiel/einfaches-gewinnspiel.php
<?php
$eingabe = 56431;
while ($eingabe !== 3545245) {
    if ($eingabe === 3545245) {
        echo "<p>Herzlichen Glückwunsch, du hast gewonnen!</p>";
        break;
    } else {
        echo "<p>Leider nichts gewonnen, versuche es erneut.</p>";
        break;
    }
}
?>